pkg/controller/master/backup/backup_target.go

```diff
@@ -129,6 +136,12 @@ func (h *TargetHandler) OnBackupTargetChange(_ string, setting *harvesterv1.Sett
 		return h.setConfiguredCondition(setting, "", fmt.Errorf("Invalid backup target type:%s or parameter", target.Type))
 	}
 
+	if target.RefreshIntervalInSeconds > 0 {
```

Comment:
When `0` is a valid value of `RefreshIntervalInSeconds`, why it only update to LH when `RefreshIntervalInSeconds>0`,

if user sets the `RefreshIntervalInSeconds` to 10, then set back to 0, what will happen?

Reply:
The default poll interval in LH is 5 minutes. We update it when `RefreshIntervalInSeconds>0` to avoid that there is data in Harvester, but no data in LH.

> if user sets the RefreshIntervalInSeconds to 10, then set back to 0, what will happen?

It's ok. LH will keep refresh. However, in Harvester, it doesn't refresh backup target next time.